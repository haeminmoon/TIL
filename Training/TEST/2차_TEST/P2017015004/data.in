100
46 16
250 7
45 8
107 32
171 32
347 15
307 73
285 58
366 32
136 66
130 86
222 26
250 4
334 97
446 79
293 42
66 66
79 30
359 16
472 61
492 72
108 39
175 92
496 96
239 20
108 36
419 2
223 72
234 33
4 3
59 23
223 6
143 80
133 88
323 58
331 50
491 18
240 48
72 26
39 17
179 94
273 90
302 48
55 51
278 40
197 67
466 93
216 13
479 22
484 95
437 16
262 23
10 4
346 85
247 54
422 17
461 50
159 11
30 28
212 46
100 54
96 34
258 44
100 23
247 6
87 83
175 34
272 88
372 39
232 28
145 87
451 6
336 77
351 91
181 63
178 19
364 62
269 17
137 99
421 90
490 13
198 98
257 88
63 6
47 29
26 13
468 18
240 57
143 72
36 17
454 28
129 27
17 16
361 91
158 34
332 24
67 56
169 86
411 16
165 77
