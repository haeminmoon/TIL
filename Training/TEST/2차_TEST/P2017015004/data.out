Case #1
554224611
Case #2
504964560
Case #3
4790071
Case #4
755252927
Case #5
880013171
Case #6
584598231
Case #7
989366279
Case #8
86398471
Case #9
625170360
Case #10
859850000
Case #11
55721231
Case #12
842828681
Case #13
635562
Case #14
867278923
Case #15
424747505
Case #16
329624334
Case #17
0
Case #18
23466407
Case #19
357297595
Case #20
557775280
Case #21
73898504
Case #22
63093257
Case #23
769037611
Case #24
94966160
Case #25
109405418
Case #26
54443544
Case #27
209
Case #28
777481211
Case #29
166969450
Case #30
1
Case #31
964269876
Case #32
715708279
Case #33
665548512
Case #34
78402934
Case #35
465328394
Case #36
935061339
Case #37
690054399
Case #38
382376916
Case #39
970519299
Case #40
308233783
Case #41
377859730
Case #42
723100840
Case #43
934565338
Case #44
6201
Case #45
798231467
Case #46
262077507
Case #47
546234360
Case #48
266459378
Case #49
481414225
Case #50
959753365
Case #51
888414705
Case #52
639514158
Case #53
22
Case #54
178342103
Case #55
201265458
Case #56
674240812
Case #57
404152212
Case #58
613335618
Case #59
15
Case #60
822724558
Case #61
650263527
Case #62
756149393
Case #63
460386643
Case #64
977694665
Case #65
201102742
Case #66
25585
Case #67
552924418
Case #68
30666976
Case #69
349891213
Case #70
871394405
Case #71
357016413
Case #72
63790465
Case #73
954228267
Case #74
642156893
Case #75
472618948
Case #76
341711751
Case #77
327926594
Case #78
247063975
Case #79
603024364
Case #80
351184938
Case #81
961024576
Case #82
455777209
Case #83
492626208
Case #84
1078507
Case #85
205278591
Case #86
400024
Case #87
942791094
Case #88
694501854
Case #89
185042843
Case #90
238819350
Case #91
313553334
Case #92
906982434
Case #93
1
Case #94
856541471
Case #95
44315247
Case #96
734474491
Case #97
180049795
Case #98
326537423
Case #99
853272440
Case #100
800440057
